test/v1/connectors/DeleteBenefitAmountsConnectorSpec.scala

```diff
@@ -31,7 +31,7 @@ class DeleteBenefitAmountsConnectorSpec extends ConnectorSpec {
 
   "DeleteBenefitAmountsConnector" should {
     "return a 200 result on delete" when {
-      "the downstream call is successful and not tax year specific" in new DesTest with Test {
+      "the downstream call is successful, not tax year specific and the desIf_Migration feature-switch is turned off" in new DesTest with Test {
         def taxYear: TaxYear                               = TaxYear.fromMtd("2017-18")
```

Comment:
Can we add:
```
        MockedAppConfig.featureSwitches returns  Configuration("desIf_Migration.enabled" -> false)
```
in here to make it more explicit what the conditions for the test are.